### ISSUE TITLE
Add support for more learning rate schedulers to Trainer class.

### ISSUE THREAD

**Issue by ecm200** [ecm200/caltech_birds#9]

**Comment by ecm200** [comment]
Commit 1b6576b404e8653a9578cb333374e946a12d9436 added all standard PyTorch rate schedulers to options.
Configuration format changed so that scheduler parameters are provided with key value pair list.
This closes this issue.
